ext/cl_khr_subgroup_extensions.asciidoc

```diff
@@ -641,7 +641,7 @@ This extension adds support for clustered reductions that operate on a subset of
 This section describes arithmetic operations that are performed on a subset of work items in a subgroup, referred to as a cluster.
 A cluster is described by a specified cluster size.
 Work items in a subgroup are assigned to clusters such that for cluster size _n_, the _n_ work items in the subgroup with the smallest subgroup local IDs are assigned to the first cluster, then the _n_ remaining work items with the smallest subgroup local IDs are assigned to the next cluster, and so on.
-The specified cluster size must be an integer constant expression that is a power-of-two.
+The specified cluster size must be an integer constant expression that is a power-of-two, otherwise the behaviour is undefined.
 Behavior is undefined if the specified cluster size is greater than the maximum size of a subgroup within the dispatch.
```

Comment:
How do you feel about rolling the clarification into this sentence instead?

```suggestion
Behavior is undefined if the specified cluster size is not an integer constant expression, is not a power-of-two, or is greater than the maximum size of a subgroup within the dispatch.
```

Reply:
Good idea, it flows better.